commands/export.go

```diff
@@ -107,12 +111,11 @@ func init() {
 }
 
 func exportCreateCmdF(c client.Client, command *cobra.Command, args []string) error {
-	var data map[string]string
-	withAttachments, _ := command.Flags().GetBool("attachments")
-	if withAttachments {
-		data = map[string]string{
-			"include_attachments": "true",
-		}
+	data := make(map[string]string)
+
+	excludeAttachments, _ := command.Flags().GetBool("no-attachments")
+	if !excludeAttachments {
```

Comment:
I think we should still support `attachment` flag, we can deprecate it but right now it has no effect.

Reply:
Sure, your comment in the JIRA ticket was to add a reverse flag. But we can have the original flag as well.

> I think we can just make including attachments by default. Exporting w/o attachments could done via extra flag/parameter.

Reply:
Yes this was I wanted, but we need to maintain the backwards compatibility for the major version at least 😅